src/clients/Queue.js

```diff
@@ -38,9 +38,10 @@ export default class Queue extends Client {
     this.listWorkerTypes.entry = {type:'function',method:'get',route:'/provisioners/<provisionerId>/worker-types',query:['continuationToken','limit'],args:['provisionerId'],name:'listWorkerTypes',stability:'experimental',output:true}; // eslint-disable-line
     this.getWorkerType.entry = {type:'function',method:'get',route:'/provisioners/<provisionerId>/worker-types/<workerType>',query:[],args:['provisionerId','workerType'],name:'getWorkerType',stability:'experimental',output:true}; // eslint-disable-line
     this.declareWorkerType.entry = {type:'function',method:'put',route:'/provisioners/<provisionerId>/worker-types/<workerType>',query:[],args:['provisionerId','workerType'],name:'declareWorkerType',stability:'experimental',scopes:[['queue:declare-worker-type:<provisionerId>/<workerType>#<property>']],input:true,output:true}; // eslint-disable-line
-    this.listWorkers.entry = {type:'function',method:'get',route:'/provisioners/<provisionerId>/worker-types/<workerType>/workers',query:['continuationToken','limit','disabled'],args:['provisionerId','workerType'],name:'listWorkers',stability:'experimental',output:true}; // eslint-disable-line
+    this.listWorkers.entry = {type:'function',method:'get',route:'/provisioners/<provisionerId>/worker-types/<workerType>/workers',query:['continuationToken','limit','quarantined'],args:['provisionerId','workerType'],name:'listWorkers',stability:'experimental',output:true}; // eslint-disable-line
```

Comment:
This looks like a breaking change, renaming the query property. @djmitche?

Reply:
The front-end has already been changed to reflect this endpoint: https://github.com/taskcluster/taskcluster-tools/blob/0c839a2718f190c6dd38029030c3a9896f2d2f54/src/views/Workers/WorkerManager.js#L115.

listWorkers will have the `quarantined` query when a user filters on quarantined workers through the filter dropdown in the workers page. In the default behaviour when loading the workers page, the `quarantined` property is not set.

Reply:
Yes, it was a breaking change to the API, but it's still experimental so nbd :)